lib/screens/settings/chat_history.dart

```diff
@@ -164,6 +165,50 @@ class ChatHistoryScreen extends StatelessWidget {
                 }
               },
             ),
+            Consumer<TwitchMessageConfig>(builder: (context, model, child) {
+              return Padding(
+                padding: const EdgeInsets.all(16),
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    Text("Pin mod message duration",
```

Comment:
More a question than a change suggestion: could it be better to use moderator instead of mod for more clarity? Could that be useful for less familiarized users?